tests/scripts/github-action-helper.sh

```diff
@@ -217,6 +217,7 @@ function deploy_cluster() {
   kubectl create -f filesystem-mirror.yaml
   kubectl create -f nfs-test.yaml
   kubectl create -f subvolumegroup.yaml
+  kubectl create -f radosnamespace.yaml
```

Comment:
In addition to creating the namespace, can the test use the namespace or validate it exists?